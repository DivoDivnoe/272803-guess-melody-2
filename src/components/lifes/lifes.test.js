import React from 'react';
import renderer from 'react-test-renderer';
import Lifes from './lifes.jsx';

describe(`Lifes component`, () => {
  it(`is rendered correctly`, () => {
    const mistakes = 1;

    const tree = renderer.create(
        <Lifes
          mistakes={mistakes}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
