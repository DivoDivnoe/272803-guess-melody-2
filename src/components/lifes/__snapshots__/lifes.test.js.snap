// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lifes component is rendered correctly 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
</div>
`;
